data/human/cim-events.txt

```diff
@@ -0,0 +1,150 @@
+			`	It doesn't look like there will be peace any time soon.`
+				decline
+
+mission "oathkeepers founded (non-FW)"
+	landing
+	to offer
+		has "event: oathkeepers founded"
+		not "Liberate Kornephoros: done" 
+	source
+		government "Free Worlds" "Republic" "Syndicate" "Independent" "Neutral" "Quarg"
+	on offer
+		log "Factions" "Oathkeepers" `The "Oathkeepers" are a regiment of Navy officers who were captured by the Free Worlds and paroled on the condition that they take no further part in fighting against the Free Worlds. They are led by William Danforth, a famous Navy admiral. Their stated mission is to protect the Paradise Worlds from the pirates who live on the anarchist worlds in the far north of human space.`
+		log "People" "William Danforth" `Danforth is a famous Navy Admiral who is a folk hero throughout human space due to his fair and honorable behavior and his willingness to use his fleet to assist people in need, even when he has to go against Parliament's orders to do so. He is currently in charge of the Navy regiment who call themselves the "Oathkeepers."`
+		conversation
+			`When you land on <origin>, it seems that everyone in the spaceport is talking about a recent news story: the Navy has just announced the formation of a new regiment, called the Oathkeepers. This regiment is staffed mostly by prisoners paroled from Kornephoros, who honor an oath to take no further part in the conflict against the Free Worlds. They will be headquartered on Farpoint, in the far north, to defend that region of space against pirate incursions.`
+			`	Furthermore, the leader of the Oathkeepers will be Admiral Danforth, an elderly admiral who is something of a folk hero and is spoken of with admiration and respect even here in Free space. Although he was not one of the paroled prisoners, he has become an outspoken advocate for a diplomatic solution to the Free Worlds conflict.`
+				decline
+
+mission "event: fw northern expansion (ScS)"
+	landing
+	to offer
+		has "event: fw northern expansion"
+		has "ScS Revenge sweet Revenge: offered"
+	source
+		government "Free Worlds" "Republic" "Syndicate" "Independent" "Neutral" "Quarg"
+	on offer
+		log `The war grows more bloody, with the Free Worlds claiming several worlds to the north of their original territory.`
+		conversation
+			`When you land on <origin>, the latest news from the war is filling the headlines. It seems the Free Worlds have occupied more systems to the North of their original territory. Whether the defecting worlds invited this or the Free Worlds annexed them against their will, seems to be a matter of some dispute.`
+
+# syndicate are normally neutral to alphas.... not sure how much this is intentional, and how much because nobody's needed to consider it. For now, hostility is set and unset: but this is risky: if main-game changes this later, thi s will override!
```

Comment:
```suggestion
# syndicate are normally neutral to alphas.... not sure how much this is intentional, and how much because nobody's needed to consider it. For now, hostility is set and unset: but this is risky: if main-game changes this later, this will override!
```
also I wouldn't do that, just make them hostile if they're going to be hostile in this campaign, if we ever want alpha to not be attacked by anyone we can just give them `marked`